ULWGL-database.csv

```diff
@@ -587,3 +587,4730 @@ Soldier of Fortune II: Double Helix - Gold Edition,gog,1228964594,ulwgl-12289645
+Yoku's Island Express,gog,1576137547,ulwgl-334940,,
+Out There: Ω Edition,gog,1958474562,ulwgl-334420,,
+Sucker for Love: First Date,gog,1716283799,ulwgl-1574270,,
+Harvester,gog,1207661373,ulwgl-287020,,
+BloodRayne 2: Terminal Cut,gog,2110288694,ulwgl-1373550,,
+Scythe: Digital Edition,gog,1621160758,ulwgl-718560,,
+NecroBouncer,gog,1988653875,ulwgl-1786830,,
+Kenshi,gog,1193046833,ulwgl-233860,,
+Xanadu Next,gog,1284895531,ulwgl-312560,,
+METAL SLUG 3,gog,1370911530,ulwgl-250180,,
+Terraforming Mars,gog,2013033253,ulwgl-800270,,
+The Smurfs 2 - The Prisoner of the Green Stone,gog,1906012961,ulwgl-2397500,,
+Bridge Constructor Medieval,gog,1633252128,ulwgl-319850,,
+Ether One Redux,gog,1207661343,ulwgl-265950,,
+Clustertruck,gog,1661530902,ulwgl-397950,,
+1954 Alcatraz,gog,1207661333,ulwgl-255280,,
+Strategic Mind: The Pacific,gog,1961554687,ulwgl-991810,,
+Medieval Dynasty,gog,1224667888,ulwgl-1129580,,
+Smelter,gog,1804464869,ulwgl-1094540,,
+Syberia: The World Before Prologue,gog,1185739490,ulwgl-1421230,,
+Might and Magic 6 - The Mandate of Heaven,gog,1207661253,ulwgl-243380,,
+Gord,gog,1986523815,ulwgl-1351210,,
+FixFox,gog,1082323622,ulwgl-1696220,,
+Ashen,gog,1117352613,ulwgl-649950,,
+Gothic 1 Remake,gog,1537831577,ulwgl-1297900,,
+Falling Frontier,gog,1128592021,ulwgl-1280190,,
+Heroes of Might and Magic V - Tribes of the East,gog,1207661193,ulwgl-15370,,
+Rail Route,gog,2011886219,ulwgl-1124180,,
+Apostle,gog,1459974775,ulwgl-1015730,,
+Crysis Warhead,gog,1666511461,ulwgl-17330,,
+We Are Not Alone,gog,2084106840,ulwgl-2249440,,
+Heroes of Might and Magic V + Hammers of Fate,gog,1207661143,ulwgl-15380,,
+The Fall,gog,1425568340,ulwgl-290770,,
+Hard West 2,gog,1345516108,ulwgl-1282410,,
+Planescape: Torment: Enhanced Edition,gog,1132393016,ulwgl-466300,,
+Not Tonight,gog,1592291898,ulwgl-733790,,
+Cave Story®+,gog,1114665528,ulwgl-200900,,
+Chroma Squad,gog,1430123043,ulwgl-251130,,
+Star Trek™: Starfleet Command Gold Edition,gog,1429172763,ulwgl-364820,,
+Tzar: The Burden of the Crown,gog,1207659252,ulwgl-825730,,
+Humans Must Answer,gog,1207659248,ulwgl-261740,,
+King's Bounty II,gog,1735160317,ulwgl-1135300,,
+They Always Run,gog,1150677504,ulwgl-1585440,,
+Fox Girls Never Play Dirty!,gog,1430778367,ulwgl-1413970,,
+Colossal Cave,gog,2090725880,ulwgl-2215540,,
+SOS OPS!,gog,2050028021,ulwgl-2475460,,
+ABZÛ,gog,1622602219,ulwgl-384190,,
+Jack Keane 2: The Fire Within,gog,1207659241,ulwgl-236970,,
+River City Girls,gog,1439166945,ulwgl-1049320,,
+Singularity™,gog,1207659239,ulwgl-42670,,
+FORCED SHOWDOWN,gog,1453126104,ulwgl-265000,,
+Below the Stone,gog,1235841495,ulwgl-1170230,,
+Steel Division 2,gog,1911255510,ulwgl-919640,,
+Jade Empire: Special Edition,gog,1207659237,ulwgl-7110,,
+SKYHILL,gog,1458729425,ulwgl-382140,,
+The Adventure Pals,gog,1391555008,ulwgl-396710,,
+Leap of Love,gog,1111421371,ulwgl-1469300,,
+Inherit the Earth,gog,1207659231,ulwgl-257260,,
+Alone in the Dark 3,gog,1207660973,ulwgl-548900,,
```

Comment:
How will we be handling DOSBox games? Are they in scope?

Reply:
I guess I can add filtering for those, each launcher will probably try to use native dosbox version

Reply:
I'd assume the same for ScummVM games? I don't think GOG uses any other tools besides DOSBox & ScummVM for releases.